projects/core/src/organization/model/search-config.ts

```diff
@@ -4,4 +4,4 @@ export interface B2BSearchConfig {
   sort?: string;
 }
 
-export const LIST = 'list';
+export const ALL = 'all';
```

Comment:
Isn't it to generic?

Reply:
we wrap ids in serialized params (query), but what if api is not parameterized? That's my propose for object name for all available data